catalogs/catalog-messaging-kafka/src/main/java/com/datastrato/gravitino/catalog/kafka/KafkaCatalogOperations.java

```diff
@@ -96,23 +144,48 @@ public NameIdentifier[] listSchemas(Namespace namespace) throws NoSuchCatalogExc
   @Override
   public Schema createSchema(NameIdentifier ident, String comment, Map<String, String> properties)
       throws NoSuchCatalogException, SchemaAlreadyExistsException {
-    throw new UnsupportedOperationException();
+    // It appears that the "default" schema suffices, so there is no need to support creating schema
+    // currently
+    throw new UnsupportedOperationException("Kafka catalog does not support schema creation");
```

Comment:
Can you please add more reasons here in the exception message, not  just an comment.